README.rst

```diff
@@ -161,20 +160,31 @@ The tests are also run automatically with `Travis CI <https://travis-ci.org/RaRe
 Supported Compression Formats
 -----------------------------
 
-``smart_open`` allows reading and writing gzip, bzip2 and xz files.
+``smart_open`` allows reading and writing gzip and bzip2 files.
 They are transparently handled over HTTP, S3, and other protocols, too, based on the extension of the file being opened.
-You can easily add support for other file extensions and compression formats:
+You can easily add support for other file extensions and compression formats.
+For example, to open xz-compressed files:
 
 .. code-block:: python
 
-    def _handle_lzma(file_obj, mode):
-        import lzma
-        return lzma.LZMAFile(filename=file_obj, mode=mode, format=lzma.FORMAT_ALONE)
-
+    import lzma, os
     from smart_open import open, register_compressor
-    register_compressor('.lzma', _handle_lzma)
-    with open('file.lzma', ...) as fin:
-        pass
+
+    def _handle_xz(file_obj, mode):
+        return lzma.LZMAFile(filename=file_obj,
```

Comment:
[Hanging indent](https://www.python.org/dev/peps/pep-0008/#id3) please. We eschew vertical-align indent altogether, for reasons of hard-to-maintain formatting and frequent mistakes.

Reply:
Done